README.md

```diff
@@ -161,6 +162,7 @@ MIT © Igor Shubovych
 [config]: https://github.com/DavidAnson/markdownlint#optionsconfig
 [style-folder]: https://github.com/DavidAnson/markdownlint/tree/main/style
 [rc-standards]: https://www.npmjs.com/package/rc#standards
+[markdownlint-cli2]: https://github.com/DavidAnson/markdownlint-cli2
```

Comment:
This list is not fully sorted, but could you move this line below "markdownlint", please?